galley/pkg/config/processor/transforms/serviceentry/pod/cache.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2019 Istio Authors
```

Comment:
Is this an infinitely-sized cache with an infinite entry TTL? I don't see replacement logic or any  timed eviction logic, or did I miss it?

Reply:
This is not cache in the actual sense. It is an in-memory lookup store. It has an event handler, which can handle delete events.

I'd like to replace this with a much better defined LookupTable model, but right now, it is simply a copy of the original code.